### ISSUE TITLE
All Modules are broken

### ISSUE THREAD

**Issue by mattlyons0** [Semantic-UI-Vue/Semantic-UI-Vue#1]
See https://semantic-ui-vue.github.io/#/modules/accordion

I am getting the following error on page load:

```
vue.esm.js:566 TypeError: Cannot read property 'map' of undefined
    at VueComponent.render (Menu.jsx:42)
    at VueComponent.Vue._render (vue.esm.js:4128)
    at VueComponent.updateComponent (vue.esm.js:2538)
    at Watcher.get (vue.esm.js:2881)
    at new Watcher (vue.esm.js:2870)
    at mountComponent (vue.esm.js:2542)
    at VueComponent.Vue$3.$mount (vue.esm.js:7863)
    at VueComponent.Vue$3.$mount (vue.esm.js:10066)
    at init (vue.esm.js:3501)
    at createComponent (vue.esm.js:5147)
```

This occurs in all modules in the docs.

Using Chromium 60 on Linux

**Comment by mariolamacchia** [comment]
@mattlyons0 thanks for submitting the issue.
The problem occurs when there are no examples available. I'll add a "No examples available" page for those cases.

**Comment by mariolamacchia** [comment]
Fixed in 0.0.9
